cfgov/unprocessed/js/organisms/FormSubmit.js

```diff
@@ -162,7 +161,7 @@ function FormSubmit( element, baseClass, opts ) {
     }
 
     function fadeInMessage() {
-      _notification.update( _notification.SUCCESS, message );
+      _notification.update( Notification.SUCCESS, message );
```

Comment:
Same issue in https://github.com/cfpb/cfgov-refresh/pull/4770. I swear I updated this but I think I rebased it wrong before merging :(